docs/querying/sql-functions.md

```diff
@@ -2541,136 +2542,411 @@ Returns the following:
 
 Adds the expression to the end of the array.
 
-* **Syntax:**: `MV_APPEND(arr1, expr)`
+* **Syntax:** `MV_APPEND(arr1, expr)`
 * **Function type:** Multi-value string
 
+<details><summary>Example</summary>
+
+The following example appends the string `label` to the multi-value string `tags` from `mvd_example`:
+
+```sql
+SELECT MV_APPEND("tags", "label") AS append
+FROM "mvd_example"
+LIMIT 1
+```
+
+Returns the following:
+
+| `append` |
+| -- |
+| `["t1","t2","t3","row1"]` |
+
+</details>
+
 [Learn more](sql-multivalue-string-functions.md)
 
 ## MV_CONCAT
 
 Concatenates two arrays.
 
-* **Syntax:**: `MV_CONCAT(arr1, arr2)`
+* **Syntax:** `MV_CONCAT(arr1, arr2)`
 * **Function type:** Multi-value string
 
+<details><summary>Example</summary>
+
+The following example concatenates `tags` from `mvd_example` to itself:
+
+```sql
+SELECT MV_CONCAT("tags", "tags") AS cat
+FROM "mvd_example"
+LIMIT 1
+```
+
+Returns the following:
+
+| `cat` |
+| -- |
+| `["t1","t2","t3","t1","t2","t3"]` |
+
+</details>
+
 [Learn more](sql-multivalue-string-functions.md)
 
 
 ## MV_CONTAINS
 
 Returns true if the expression is in the array, false otherwise.
 
-* **Syntax:**: `MV_CONTAINS(arr, expr)`
+* **Syntax:** `MV_CONTAINS(arr, expr)`
 * **Function type:** Multi-value string
 
+<details><summary>Example</summary>
+
+The following example checks whether the string `t3` is located within `tags` from `mvd_example`:
+
+```sql
+SELECT MV_CONTAINS("tags", 't3') AS contained
+FROM "mvd_example"
+LIMIT 1
```

Comment:
The following query might be a bit more illustrative since it shows the row & the value and includes both true & false:

```
SELECT tags, MV_CONTAINS("tags", 't3') AS contained
FROM "mvd_example"
```